src/test/e2e/00_use_cli_test.go

```diff
@@ -116,6 +116,26 @@ func TestUseCLI(t *testing.T) {
 		require.Error(t, err, stdOut, stdErr)
 	})
 
+	t.Run("zarf package create for archive path", func(t *testing.T) {
+		t.Parallel()
+		stdOut, stdErr, err := e2e.Zarf("package", "create", "packages/distros/eks", "--zarf-cache", "--confirm")
+		require.NoError(t, err, stdOut, stdErr)
+	})
+
+	t.Run("zarf package inspect for archive path", func(t *testing.T) {
+		t.Parallel()
+		path := "build/zarf-package-distro-eks-multi-0.0.2.tar.zst"
+		stdOut, stdErr, err := e2e.Zarf("package", "inspect", path)
+		require.NoError(t, err, stdOut, stdErr)
+	})
+
+	t.Run("zarf package deploy for archive path", func(t *testing.T) {
+		t.Parallel()
+		path := "build/zarf-package-distro-eks-multi-0.0.2.tar.zst"
+		stdOut, stdErr, err := e2e.Zarf("package", "deploy", path, "--confirm")
+		require.NoError(t, err, stdOut, stdErr)
```

Comment:
This should check that the right `eksctl` is spit out and should clean up the file when done.

Reply:
https://github.com/defenseunicorns/zarf/blob/314e07cd895ecbd2d9deaa2185e6b4a6843355af/packages/distros/eks/zarf.yaml#L29 the package's actions delete eksctl